TrajectoryPlannerROS:
  min_vel_x: 0.1
  max_vel_x: 0.3
  min_vel_theta: -1.0
  max_vel_theta: 1.0
  min_in_place_vel_theta: 0.4

  acc_lim_x: 2.5
  acc_lim_y: 0.0
  acc_lim_theta: 3.2

  holonomic_robot: false
  meter_scoring: true
